content/developers/dogstatsd.md

```diff
@@ -50,9 +50,17 @@ where it will be stored and available for graphing alongside the rest of your me
 
 ## Setup
 
-Once you have the Datadog Agent installed on your application servers/containers—or anywhere
-your application can reliably reach—grab the [DogStatsD client library][2] for your
-application language and you'll be ready to start hacking. You _can_ use any generic StatsD client to send metrics to DogStatsD, but you won't be able to use any of the Datadog-specific features mentioned above.
+First, take a look at your `datadog.conf` file, and uncommented the following lines:
+``` 
+# If you don't want to enable the DogStatsd server, set this option to no
```

Comment:
wait, uncomment this? If not, don't include this line here

Reply:
👍